cristin-import/src/test/java/cucumber/ReportFeatures.java

```diff
@@ -37,7 +37,7 @@ public void thatTheCristinResultHasAnEmptyPublisherNameField() {
     }
 
     @Then("the NVA Resource Report has a Publisher that cannot be verified through a URI")
-    public void theNVAResourceReportHasAPublisherThatCannotBeVerifiedThroughAUri() {
+    public void theNvaResourceReportHasAPublisherThatCannotBeVerifiedThroughAUri() {
```

Comment:
fix the name because checkstyle complains

Reply:
Funnny it accepts "AUri"